manifests/stage-macvlan-cni/0010-macvlan-ds.yml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 NVIDIA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: macvlan-cni-ds
```

Comment:
I don't think we want to maintain images per generic cni. 
Another option is just to install containernetworking-plugins

Reply:
This discussion should be on PR#37 IMO :)

Reply:
So here is what i propose,

rename the stage to be : `stage-containernetworking-plugins` which will deploy a daemonset that will install (cp to /opt/cni/bin on host) the provided cni plugins.

it means we build one image for all plugins based on:
binaries published : https://github.com/containernetworking/plugins/releases

i prefer not to rely on distro specific repos

@moshe010 WDYT ?